docs/source/api/cachegroupparameters.rst

```diff
@@ -97,6 +105,32 @@ The request data can take the form of either a single object or an array of one
 		"parameterId": 124
 	}
 
+Array Format
+""""""""""""
+
+:cacheGroupId: An integer that is the :ref:`cache-group-id` of the :term:`Cache Group` to which a :term:`Parameter` is being assigned
+:parameterId:  An integer that is the :ref:`parameter-id` of the :term:`Parameter` being assigned
+
+.. code-block:: http
+	:caption: Request Example
+
+	POST /api/1.1/cachegroupparameters HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+	Content-Length: 39
+	Content-Type: application/json
+
+	[{
+		"cachegroupId": 8,
```

Comment:
this and below need to be `cacheGroupId`

Reply:
done